static/js/chart/draw.ts

```diff
@@ -600,20 +616,24 @@ function drawGroupLineChart(
         new Date(dp.label).getTime(),
         dp.value,
       ]);
-
       const line = d3
         .line()
         .x((d) => xScale(d[0]))
         .y((d) => yScale(d[1]));
-
+      let color: string;
```

Comment:
The split in logic here between computePlotParams and for the callers suggests that the interface might need to change. Perhaps computePlotParams should return a closure that takes in two params (place, statvar) and returns the right dash and color.

Reply:
Fixed